src/test/java/com/greedobank/reports/mapper/NewsMapperTest.java

```diff
@@ -0,0 +1,80 @@
+package com.greedobank.reports.mapper;
+
+import com.greedobank.reports.dto.ContentRequestDTO;
+import com.greedobank.reports.dto.ContentResponseDTO;
+import com.greedobank.reports.dto.NewsRequestDTO;
+import com.greedobank.reports.dto.NewsResponseDTO;
+import com.greedobank.reports.model.News;
+import lombok.val;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.internal.matchers.apachecommons.ReflectionEquals;
+
+import java.time.OffsetDateTime;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class NewsMapperTest {
+
+    private NewsMapper mapper;
+
+    @BeforeEach
+    private void setUp() {
+        mapper = new NewsMapper();
+    }
+
+    @Test
+    void toNewsFromRequest() {
+        NewsRequestDTO request = new NewsRequestDTO(
+                true,
```

Comment:
ohhh, this looks so long:) maybe some builder/accessors chain/smth else may help